site/src/ThemeTypeLayer.jsx

```diff
@@ -141,6 +141,22 @@ const ThemeTypeLayer = ({
           {...(minzoom ? { minzoom } : {})}
         />
       ) : null}
+      {outline ? (
+        <Layer
```

Comment:
Add an invisible line which is larger than the rendered outline to enable easier clicks.